gen/definitions/properties.yaml

```diff
@@ -186,6 +186,8 @@ pkiTP:
     parent_dn: "uni/userext/pkiext"
   overwrites:
     cert_chain: "certificate_chain"
+  ignores:
```

Comment:
why are we ignoring this?

Reply:
The next time the re-gen metadata workflow runs it will find this new property and currently the generator is throwing an error. I am working on fixing it in a follow up PR just for pkiTP.